lib/declare_schema/model.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'rails'
```

Comment:
Do we need to require all of `rails` here? Or is it only `activerecord` that we need to require?

Reply:
It's there so we can check the `Rails::VERSION`. Other parts of this repo do the same thing, so this isn't add another dependency. It's just allowing this test to run in isolation.

Reply:
Once we drop support for Rails 4, we can remove all these special cases for < 5.

Reply:
Sounds good, thanks for the clarification.